include/itkCuberilleImageToMeshFilter.hxx

```diff
@@ -247,12 +255,13 @@ CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>
 }
 
 template<typename TInputImage, typename TOutputMesh, typename TInterpolator>
-void
+typename CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>::PointVectorType
 CuberilleImageToMeshFilter<TInputImage,TOutputMesh,TInterpolator>
 ::AddVertex( typename TOutputMesh::PointIdentifier &id,
              typename TInputImage::IndexType index,
              const TInputImage* image,
-             TOutputMesh* mesh )
+             TOutputMesh* mesh,
+             const size_t num_components )
```

Comment:
ITK's style guide call for `camelCase`, and no `names_with_underscores`. It appears in a few more places in this PR.